site/en/configure/coverage.md

```diff
@@ -188,8 +188,60 @@ py_test(
     ],
 )
 ```
-<!-- TODO: Allow specifying a target for `PYTHON_COVERAGE`, instead of having to use `$(location)` -->
 
+If you are using a hermetic python toolchain, instead of adding an attribute to
+every `py_test` rule you can instead add the coverage tool to the toolchain
+configuration.
+
+Because the `pip_install` rule depends on the python toolchain, you cannot use
```

Comment:
I'm not familiar with pip_install, can you explain more?
Maybe this is avoidable with some adjustments somewhere.

Reply:
I could add a link to the docs for that rule.  But `pip` uses `python`, and it finds it via the toolchain, but it can't do that if you haven't configured the python toolchain yet.